src/helpers/label.ts

```diff
@@ -53,3 +53,47 @@ export const getLabel = async (name: string): Promise<boolean> => {
 
   return false;
 };
+
+// Function to update labels based on the base rate difference
+export async function updateLabelsFromBaseRate(owner: string, repo: string, labels: Label[], baseRateDifference: number) {
+  const context = getBotContext();
+  try {
+    for (const label of labels) {
+      if (label.name.startsWith("Price: ")) {
+        const labelParts = label.name.split(": ");
+        const prefix = labelParts[0];
+        const valueAndSuffix = labelParts[1].split(" ");
+
+        let labelValue = parseFloat(valueAndSuffix[0]);
+
+        let updatedLabelValue =
```

Comment:
@devpanther perhaps it is simpler to calculate label prices from scratch without calculating the difference

Reply:
> @devpanther perhaps it is simpler to calculate label prices from scratch without calculating the difference

how do you mean from scratch

Reply:
you can use an [existing function](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/shared/pricing.ts#L3) to calculate an updated price

Reply:
> you can use an [existing function](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/shared/pricing.ts#L3) to calculate an updated price

Doesn't it need an updated config to give the correct prices?

Reply:
Yes it does. As far as I understand we already have an updated config on push event so we can recalculate price labels based on it.

So on push event we:
1. check whether `base-multiplier` has been updated
2. at this point we have access to the updated config so we can recalculate price labels

Reply:
> Yes it does. As far as I understand we already have an updated config on push event so we can recalculate price labels based on it.
> 
> So on push event we:
> 
> 1. check whether `base-multiplier` has been updated
> 2. at this point we have access to the updated config so we can recalculate price labels

Yea, I saw the problem.. i was still using the camel case config